content/security.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Security"
+---
+
+After reading this guide, you'll know:
+
+1. The security surface area of a Meteor app
+2. How to secure methods, publications, and source code
+3. Where to store secret keys in development and production
+4. How to follow a security checklist when auditing your app
+
+## Security
+
+Securing a web application is all about understanding security domains and understanding the attack surface between these domains. In a Meteor app, things are pretty simple:
+
+1. Code that runs on the server can be trusted
+2. Everything else: code that runs on the client, data sent through method and publication arguments, etc, can't be trusted
+
+In practice, this means that you should do most of your security and validation on the boundary between these two domains. In simple terms:
+
+1. Validate and check all data that comes from the client
+2. Don't leak any secret data to the client
```

Comment:
Maybe "secret information" -- doesn't all have to be in the db (e.g. keys), which is what the word "data" implies